lib/ziti-tunnel-cbs/include/ziti/ziti_tunnel_cbs.h

```diff
@@ -307,6 +308,14 @@ struct add_identity_request_s {
     void *cmd_ctx;
 };
 
+#ifndef IPC_SUCCESS
+#define IPC_SUCCESS 0
+#endif
+#ifndef IPC_ERROR
+#define IPC_ERROR 500
```

Comment:
500 feels like these are maybe sorta trying to be like http status codes? If so should IPC_SUCCESS be 200?

Reply:
Currently, there are different error codes defined for different types of error. These are not similar to http error codes. However, UI is checking for only 0 (success) and 500(error) error codes, so I copied only those 2 error codes in to t-sdk.